docker/README.md

```diff
@@ -1,20 +1,58 @@
 # Build in Docker
 
-We provide a Dockerfile to build the project in a container.
+We provide the following Dockerfiles:
+- [Dockerfile](./Dockerfile) - for building the Scala API.
+- [Dockerfile.python](./Dockerfile.python) - for building the Python API (using conda for RAPIDS dependencies).
+- [Dockerfile.pip](./Dockerfile.pip) - for building the Python API (using pip for RAPIDS dependencies).
 
-## Build the container
+## Scala API
 
-```bash
-docker build -t rapids-ml:latest -f Dockerfile .
-```
-Please check the [Dockerfile](./Dockerfile) for more configurable build arguments.
+- Build the container.
+  ```bash
+  # cd spark-rapids-ml/docker
```

Comment:
Maybe explain the rationale for the commented lines in the various example commands.

Reply:
Tried to clarify by making all instructions just relative to this `docker` folder.